### PR TITLE
Improve interface of homology classes

### PR DESCRIPTION
**Pull request by saraedum**
Checklist
* [x] Added an entry in `doc/news/`. 
* [x] Added a test for this change.
* [x] Added new .py files to the documentation in `doc/geometry` or `doc/graphical`. **Nothing new.**

### CODE REVIEW

File: flatsurf/geometry/homology.py [hunk 1]
```diff
@@ -361,6 +377,64 @@ def coefficient(self, gen):
...
+            sage: H = SimplicialHomology(T)
+            sage: a, b = H.gens()
+            sage: (a - b).simplex_coefficients()
+            ((1, (0, 1)), (-1, (0, 0)))
+
+        """
+        return tuple((self._chain[s], s) for s in self._chain.support())
+
+    @cached_method
+    def holonomy(self):
```
Comment (videlec) [inline flatsurf/geometry/homology.py:399]: I do not think it is a good idea to have this at the level of homology. More precisely, homology makes sense for a topological surface (even though right now it belongs to the similarity surface category). But the holonomy is only valid for translation surfaces (which is a sub category of similarity surfaces).

What would be more natural is `surface.holonomy_morphism(domain)`.

File: flatsurf/geometry/homology.py [hunk 2]
```diff
@@ -361,6 +377,64 @@ def coefficient(self, gen):
 
         return self.coefficients()[index]
 
+    @cached_method
+    def simplex_coefficients(self):
```
Comment (videlec) [inline flatsurf/geometry/homology.py:381]: I think that
```
sage: (a - b)._chain.monomial_coefficients()
{(0, 1): 1, (0, 0): -1}
```
does the job already.

File: flatsurf/geometry/homology.py (hunk 1 above)
Comment (saraedum) [reply flatsurf/geometry/homology.py:399]: Ok. Then I'll just add it as an example to `chain()`.

File: flatsurf/geometry/homology.py (hunk 2 above)
Comment (saraedum) [reply flatsurf/geometry/homology.py:381]: Ok. I'll then just add this as an example to `chain()`.

**Pull request merged**